src/ServiceInsight/MessageList/MessageListView.xaml.cs

```diff
@@ -56,7 +56,11 @@ void OnBeforeLayoutRefresh(object sender, CancelRoutedEventArgs e)
 
         void SortData(ColumnBase column, ListSortDirection order)
         {
-            Model.RefreshMessages(column.Tag as string, order == ListSortDirection.Ascending);
+            var orderBy = column.Tag as string;
+            var ascending = order == ListSortDirection.Ascending;
+
+            Model.ChangeSorting(orderBy, ascending);
+            Model.RefreshMessages();
```

Comment:
_Continuing from https://github.com/Particular/ServiceInsight/pull/737/files#r133939640_

Given this is the only call to `ChangeSorting`, i.e. it is never called without then calling `RefreshMessages`, would it be better to encaspulate that behind a `RefreshMessages` overload with `orderBy` and `ascending` parameters?